tests/test_useq_core_widgets.py

```diff
@@ -9,10 +10,11 @@
 from qtpy.QtCore import QTimer
 from qtpy.QtWidgets import QMessageBox
 
+from pymmcore_widgets._util import _get_next_available_paths
 from pymmcore_widgets.mda import MDAWidget
 from pymmcore_widgets.mda._core_channels import CoreConnectedChannelTable
 from pymmcore_widgets.mda._core_grid import CoreConnectedGridPlanWidget
-from pymmcore_widgets.mda._core_mda import CoreMDATabs
+from pymmcore_widgets.mda._core_mda import CoreMDATabs, SaveAs
```

Comment:
i believe `SaveAs` is gone?

Reply:
yes it is,  I'm working on updating the test 👍🏻